api/views.py

```diff
@@ -52,17 +52,6 @@ class DashBoardView(TemplateView):
     template_name = 'index.html'
 
 
-class HomeView(TemplateView):
```

Comment:
How are we going to serve the homepage view? 
I don't see anywhere you made provision for that.

Reply:
Resolved!